deploy/hackathon-docker/docker-compose.yml

```diff
@@ -0,0 +1,25 @@
+version: '3'
+services:
+  web:
+    image: "kaiyuanshe/hackathon:latest"
+    build:
+      context: ../../
+      dockerfile: deploy/hackathon-docker/Dockerfile
+    ports:
+      - "9988:80"
+    #command: python /opt/open-hackathon/open-hackathon-server/src/run.py
+    environment:
+      - DB_SERVER=db
+      - DB_PORT=27017
+    links:
+      - 'db'
+    volumes:
+      - "./logs:/var/log/open-hackathon"
+    container_name: "kaiyuanshe-hackathon"
+  db:
+    image: "mongo"
+    ports:
+     - "27017:27017"
+    volumes:
```

Comment:
Is this volume persistent?  Hopes it's not destroyed by K8S when the container restarts.

Reply:
We can use PV in Kube, I will write kube yaml when dockerfile is done.

Reply:
so, we can keep dockerfile the way it looks now, cool